src/clj/game/cards-assets.clj

```diff
@@ -428,6 +428,12 @@
                                          (shuffle! state side :deck))}
                            card nil))}]}
 
+   "Jeeves Model Bioroids"
+   {:abilities [{:label "Gain [Click]"
+                 :req (req (< 2 (- (:click-per-turn corp) (:click corp))))
```

Comment:
Subtracting clicks from max clicks isn't a good universal method to get spent clicks. This causes problems with i.e. Biotic Labor. Example situation where it fails: I have 3 max clicks, I play Biotic, Shipment from SanSan - I should be able to get a click back from Jeeves now to be able to play second Shipment from SanSan, but with the `req` above I can't. (This could be a nice unit test ;-) )